omnigibson/action_primitives/starter_semantic_action_primitives.py

```diff
@@ -164,14 +165,17 @@ def _assemble_robot_copy(self):
         if m.TIAGO_TORSO_FIXED:
             assert self.arm == "left", "Fixed torso mode only supports left arm!"
             joint_control_idx = self.robot.arm_control_idx["left"]
-            joint_pos = th.tensor(self.robot.get_joint_positions()[joint_control_idx])
+            joint_pos = th.tensor(self.robot.get_joint_positions()[joint_control_idx], dtype=th.float32)
```

Comment:
Shouldnt need to call th.tensor at all here and below, it should already be a tensor